lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -207,13 +227,18 @@ private void migrateIdentity(
             ReportUserIdentityItem reportItem, PageSummary pageSummary, String batchId) {
         // Skip any non P2 identities
         if (!Vot.P2.name().equals(reportItem.getIdentity())) {
-            // Send skipped audit event
             LOGGER.info(
                     annotateLog(
                             "Skipping migration - not a P2 identity",
                             reportItem,
                             batchId,
                             pageSummary));
+            auditService.sendAuditEvent(
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_EVCS_MIGRATION_SKIPPED,
+                            batchId,
```

Comment:
Think you need the component ID here intead of batchId

Reply:
Thanks forgot to replace :)